orgs-using-d.dd

```diff
@@ -150,6 +150,13 @@ $(DIVC orgs-using-d center,
             $(FA_HIRING www.weka.io)
         )
     )
+    $(DORG LLC CERERIS, http://www.myriad-corp.com, cereris.png,
```

Comment:
from the top comment: "Organizations are sorted alphabetically" - no need to add yourself to the end ;-)